packages/open_aea/connections/p2p_libp2p/libp2p_node/utils/utils.go

```diff
@@ -454,6 +475,18 @@ func KeyPairFromFetchAIKey(key string) (crypto.PrivKey, crypto.PubKey, error) {
 	return prvKey, pubKey, nil
 }
 
+// KeyPairFromEthereumKey key pair from hex encoded secp256k1 private key
+func KeyPairFromEthereumKey(key string) (crypto.PrivKey, crypto.PubKey, error) {
+	privateKey, err := ethCrypto.HexToECDSA(key[2:]) // slice off the "0x"
+	if err != nil {
+		logger.Error().Msg("Cannot encode ETH hexadecimal key to PrivateKey")
+		return nil, nil, err
+	}
+	privKey, pubKey, err := crypto.ECDSAKeyPairFromKey(privateKey)
+	return privKey, pubKey, err
```

Comment:
Why do we need to store  those variables? Cant we just `return crypto.ECDSAKeyPairFromKey(privateKey)`?